app/utility/oligrapher.rb

```diff
@@ -20,7 +20,12 @@ def self.configuration(map:, current_user: nil)
         list: JSON.parse(map.annotations_data || "[]"),
         sources: map.sources_annotation
       },
-      settings: { debug: Rails.env.development? },
+      settings: {
+        debug: Rails.env.development?,
+        embed: embed,
+        logoUrl: "https://dfl6orqdcqt4f.cloudfront.net/assets/lilsis-logo-trans-200-74169fd94db9637c31388ad2060b48720f94450b40c45c23a3889cf480f02c52.png",
```

Comment:
That's an old url (I didn't even think any of the cloudfront links worked).

You can use the path helpers:  `ActionController::Base.helpers.asset_path("lilsis-logo-trans-200.png")`